modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZonesUtil.java

```diff
@@ -248,7 +271,7 @@ static Update updateLogicalTopologyAndVersion(Set<String> logicalTopology, long
      * @return Returns a set of data nodes retrieved from data nodes map, which value is more than 0.
      */
     public static Set<String> dataNodes(Map<String, Integer> dataNodesMap) {
-        return dataNodesMap.entrySet().stream().filter(e -> e.getValue() > 0).map(Entry::getKey).collect(Collectors.toSet());
+        return dataNodesMap.entrySet().stream().filter(e -> e.getValue() > 0).map(Map.Entry::getKey).collect(Collectors.toSet());
```

Comment:
Let's restore the import java.util.Map.Entry;

Reply:
Discussed in person.